micro-ui/web/micro-ui-internals/packages/modules/core/src/App.js

```
@@ -90,12 +90,16 @@ export const DigitAppWrapper = ({ stateCode, modules, appTenants, logoUrl, initD
   const { stateInfo } = storeData || {};
   const userScreensExempted = ["user/profile", "user/error"];
   const isUserProfile = userScreensExempted.some((url) => location?.pathname?.includes(url));
+  const userDetails = Digit.UserService.getUser();
+  let CITIZEN = userDetails?.info?.type === "CITIZEN" || !window.location.pathname.split("/").includes("employee") ? true : false;
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)